testing/rabbitmq-server/APKBUILD

```diff
@@ -1,29 +1,29 @@
 # Contributor: Nathan Johnson <nathan@nathanjohnson.info>
 # Maintainer: Nathan Johnson <nathan@nathanjohnson.info>
 pkgname=rabbitmq-server
-pkgver=3.6.10
+pkgver=3.7.5
 _realver=${pkgver//\./_}
 pkgrel=0
 pkgdesc="RabbitMQ is an open source multi-protocol messaging broker."
 url="http://www.rabbitmq.com/"
 arch="noarch"
-license="MPL-1.1"
+license="MPL 1.1"
```

Comment:
Please revert this line change since aports uses the unmodified [SPDX License identifier](https://spdx.org/licenses/)